posthog/models/organization.py

```diff
@@ -87,6 +87,7 @@ class PluginsAccessLevel(models.IntegerChoices):
     )
     available_features = ArrayField(models.CharField(max_length=64, blank=False), blank=True, default=list)
     for_internal_metrics: models.BooleanField = models.BooleanField(default=False)
+    is_member_join_email_enabled: models.BooleanField = models.BooleanField(default=True)
```

Comment:
This definitely works but I wonder if it's worth doing something more abstract/reusable, a model to record notification preferences for different types of campaigns (I introduced this in a draft PR but never actually finalized it), as we'll probably have more of these soon.

It's fine to have this for now, but to avoid having another deprecated attribute, we might go a step further already?

Reply:
I'm not sure if it's the ideal thing to abstract into its own model, because some such settings will be org-level (like this one), but others project-level or user-level – and this is very lightweight to store in the org/project/user row anyway.

Reply:
Well an abstract model could handle that too, but it's fine we can do it when we add more email notifications like this